### ISSUE TITLE
PhasePlots don't correctly render ion fields for z_axis label

### ISSUE THREAD

**Issue by chummels** [yt-project/yt#3269]
### Bug report

**Bug summary**

Usage of ion fields such as `H_p0_mass` (for H I mass) are not rendered properly for `PhasePlot` z-axis labels, but they work correctly for `ProjectionPlot`s z-axis labels.  I'm happy to fix this, but I cannot find where in the code `ProjectionPlot` is doing this step.  Suggestions are welcome.

**Code for reproduction**

```python
import yt
ds = yt.load_sample('IsolatedGalaxy')
yt.PhasePlot(ds, ('gas', 'density'), ('gas', 'temperature'), ('gas', 'H_p0_mass')).save()
yt.ProjectionPlot(ds, 'x', ('gas', 'H_p0_number_density')).save()
```

**Actual outcome**

![galaxy0030_2d-Profile_density_temperature_H_p0_mass](https://user-images.githubusercontent.com/8250999/118381594-321c2700-b5a1-11eb-9e5b-c3efe59b8dee.png)

![galaxy0030_Projection_x_H_p0_number_density](https://user-images.githubusercontent.com/8250999/118381589-27fa2880-b5a1-11eb-9f58-839c5940df84.png)

**Comment by neutrinoceros** [comment]
It seems that this was fixed. Running your example again I get the expected result
![galaxy0030_2d-Profile_density_temperature_H_p0_mass](https://user-images.githubusercontent.com/14075922/186029181-a49634ed-6994-48cd-8db8-fdd0aa209b93.png)

**Comment by chummels** [comment]
I think this got fixed in #3282.
